unit-testing-react-what-why-how/Button/problem/src/styles.css

```diff
@@ -1,84 +1,47 @@
 :root {
-  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
-  line-height: 1.5;
-  font-weight: 400;
+	font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+	line-height: 1.5;
+	font-weight: 400;
 }
 
 #root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
-  display: flex;
-  flex-direction: column;
+	max-width: 1280px;
+	margin: 0 auto;
+	padding: 2rem;
+	text-align: center;
+	display: flex;
+	flex-direction: column;
 }
 
 body {
-  margin: 0;
-  display: flex;
-  min-width: 320px;
-  min-height: 100vh;
-}
-
-table {
-  border-collapse: collapse;
-  border-spacing: 0;
-  border: 1px solid #eee;
-}
-
-tr {
-  border-bottom: 1px solid #eee;
-  height: 40px;
-}
-tr:nth-child(even) {
-  background-color: #fafafa;
-}
-
-th,
-td {
-  padding: 0 20px;
-}
-
-th:first-child,
-td:first-child {
-  width: 120px;
-  text-align: left;
-}
-
-th:nth-child(2),
-td:nth-child(2) {
-  width: 50px;
-  text-align: right;
-}
-
-th:nth-child(3),
-td:nth-child(3) {
-  width: 180px;
-  text-align: left;
+	margin: 0;
+	display: flex;
+	min-width: 320px;
+	min-height: 100vh;
 }
 
 button {
-  border: 0;
-  color: white;
-  background-color: #13b0ff;
-  padding: 10px 20px;
-  border-radius: 4px;
-  cursor: pointer;
-  transition: background-color 0.2s ease;
-  margin: 10px;
+	border: 0;
+	color: white;
+	background-color: #13b0ff;
+	padding: 10px 20px;
+	border-radius: 4px;
+	cursor: pointer;
+	transition: background-color 0.2s ease;
+	margin: 10px;
 }
 button:hover {
-  background-color: #1199e6;
+	background-color: #1199e6;
 }
 button:active {
-  background-color: #0c7ebd;
-  color: #e6e6e6;
-  box-shadow: inset 0 0 10px rgba(0, 0, 0, 0.4);
+	background-color: #0c7ebd;
+	color: #e6e6e6;
+	box-shadow: inset 0 0 10px rgba(0, 0, 0, 0.4);
 }
 
 .error {
-  color: #ff4136;
+	color: #ff4136;
 }
 .loading {
-  color: #d8d8d8;
+	color: #d8d8d8;
```

Comment:
Are these classes used?

Reply:
No, they were remnants from a the exercise this was based off of. I'll remove them in the next commit, both from solution and problem code